opm/parser/share/keywords/000_Eclipse100/C/COMPSEGS

```diff
@@ -9,6 +9,6 @@
  {"name" : "DIRECTION" , "value_type" : "STRING"},
  {"name" : "END_IJK" , "value_type" : "INT"},
  {"name" : "CENTER_DEPTH" , "value_type" : "DOUBLE" , "dimension" : "Length" , "default" : 0},
- {"name" : "THERMAL_LENGTH" , "value_type" : "DOUBLE" , "dimension" : "Length" , "default" : 0},
+ {"name" : "THERMAL_LENGTH" , "value_type" : "DOUBLE" , "dimension" : "Length"},
  {"name" : "SEGMENT_NUMBER" , "value_type" : "INT"}
- ]]}
```

Comment:
> I have no idea about how things work here. Is it something related to the editor used?

Yes probably - what has happened is that the line: `]]}` is replaced with `]]}\n` - i.e. with a final newline - that is OK!